tests/unittests/common/sqlitedatabasetest.cpp

```diff
@@ -247,8 +246,13 @@ TEST_F(SQLiteDatabaseTest, testConcurrentAccessFromMultipleThreads) {
         startWorkerThread(&pool, READING | TRANSACTION, 10000);
     QFuture<WorkerResult> r3 =
         startWorkerThread(&pool, READING | NO_TRANSACTION, 10000);
+#if (!defined(Q_OS_WIN32)) && (!defined(Q_OS_WIN64))
+    // 4 threads seem to be too much for unreliable operating systems. Thus
+    // on unreliable, less known, rarely used niche operating systems we only
+    // use 3 reader threads...
```

Comment:
So much cynicism in 3 lines :wink:

Reply:
Yep, after many hours of struggling with such annoying Windows issues, I have to calm down somehow and it turned out writing such comments helps to do so :grin: